province_mapping.csv

```diff
@@ -1,541 +1,80 @@
+Samut Songkhram,สมุทรสงคราม,193305.0,414.0,467.0,Samut Songkhram,SKM,TH-75,TH54,"['SKM', 'Samut Songkhram', 'SamutSongkhram', 'สมทุรสงคราม', 'สมทุรสงครำม', 'สมุทรสงคราม', 'สมุุทรสงคราม', 'สส']",5
+Saraburi,สระบุรี,645911.0,3499.0,185.0,Saraburi,SRI,TH-19,TH37,"['SRI', 'Saraburi', 'สบ', 'สระบรุ', 'สระบุร', 'สระบุรี', 'สระบุุรี']",4
+Satun,สตูล,323586.0,3019.0,107.0,Satun,STN,TH-91,TH67,"['STN', 'Satun', 'สต', 'สตลู', 'สตูล']",12
+Sing Buri,สิงห์บุรี,208446.0,817.0,255.0,Sing Buri,SBR,TH-17,TH33,"['SBR', 'Sing Buri', 'SingBuri', 'สงิหบ์รุ', 'สงิหบ์ุร', 'สห', 'สิงห์บุร', 'สิงห์บุรี']",4
+Sisaket,ศรีสะเกษ,1472859.0,8936.0,165.0,Sisaket,SSK,TH-33,TH30,"['SSK', 'Si Sa Ket', 'SiSaKet', 'Sisaket', 'ศก', 'ศรสีะเกษ', 'ศรีสะเกษ']",10
+Songkhla,สงขลา,1435968.0,7741.0,186.0,Songkhla,SKA,TH-90,TH68,"['SKA', 'Songkhla', 'สข', 'สงขลา', 'สงขลำ']",12
+Sukhothai,สุโขทัย,595072.0,6671.0,89.0,Sukhothai (Sukhothai Thani),STI,TH-64,TH09,"['STI', 'Sukhothai', 'สท', 'สุโขทัย', 'สโุขทยั', 'สโุขทัย']",2
+Suphan Buri,สุพรรณบุรี,846334.0,5410.0,156.0,Suphan Buri,SPB,TH-72,TH51,"['SPB', 'Suphan Buri', 'SuphanBuri', 'Suphanburi', 'สพ', 'สพุรรณบรุ', 'สพุรรณบรุี', 'สพุรรณบุร', 'สุพรรณบุร', 'สุพรรณบุรี']",5
+Surat Thani,สุราษฎร์ธานี,1068010.0,13079.0,81.0,Surat Thani,SNI,TH-84,TH60,"['SNI', 'Surat Thani', 'SuratThani', 'สฎ', 'สรุาษฎรธ์าน', 'สรุาษฎรธ์านี', 'สรุำษฎรธ์ำน', 'สุราษฎร์ธาน', 'สุราษฎร์ธานี']",11
+Surin,สุรินทร์,1396831.0,8854.0,157.0,Surin,SRN,TH-32,TH29,"['SRN', 'Surin', 'สร', 'สรุนิทร', 'สุรินทร', 'สุรินทร์']",9
+Tak,ตาก,665620.0,17303.0,39.0,Tak,TAK,TH-63,TH08,"['TAK', 'Tak', 'ตก', 'ตาก', 'ตำก']",2
+Trang,ตรัง,643164.0,4726.0,136.0,Trang,TRG,TH-92,TH65,"['TRG', 'Trang', 'ตง', 'ตรงั', 'ตรัง']",12
+Trat,ตราด,229958.0,2866.0,78.0,Trat,TRT,TH-23,TH49,"['TRT', 'Trat', 'ตร', 'ตราด', 'ตรำด']",6
+Ubon Ratchathani,อุบลราชธานี,1878146.0,15626.0,120.0,Ubon Ratchathani,UBN,TH-34,TH75,"['UBN', 'Ubon Ratchathani', 'UbonRatchathani', 'อบ', 'อบุลราชธาน', 'อบุลราชธานี', 'อบุลรำชธำน', 'อุบลราชธาน', 'อุบลราชธานี']",10
+Udon Thani,อุดรธานี,1586646.0,11072.0,143.0,Udon Thani,UDN,TH-41,TH76,"['UDN', 'Ubon Thani', 'Udon Thani', 'UdonThani', 'อด', 'อดุรธาน', 'อดุรธานี', 'อดุรธำน', 'อุดรธาน', 'อุดรธานี']",8
+Uthai Thani,อุทัยธานี,328618.0,6647.0,50.0,Uthai Thani,UTI,TH-61,TH15,"['UTI', 'Uthai Thani', 'UthaiThani', 'อทัุยธาน', 'อทุยัธาน', 'อทุยัธานี', 'อทุัยธาน', 'อน', 'อุทัยธาน', 'อุทัยธานี']",3
+Uttaradit,อุตรดิตถ์,453103.0,7906.0,58.0,Uttaradit,UTD,TH-53,TH10,"['UTT', 'Uttaradit', 'อต', 'อตุรดติถ', 'อุตรดิตถ', 'อุตรดิตถ์']",2
+Yala,ยะลา,536330.0,4476.0,119.0,Yala,YLA,TH-95,TH70,"['YLA', 'Yala', 'ยล', 'ยะลา']",12
+Yasothon,ยโสธร,537299.0,4131.0,130.0,Yasothon,YST,TH-35,TH72,"['YST', 'Yasothon', 'ยส', 'ยโสธร']",10
+Prison,เรือนจำฯ,,,,,,,,"['Prison', 'เรอืนจา/ทีต่อ้งขงั', 'เรอืนจาฯ', 'เรอืนจาและทีต่อ้งขงั']",Unknown
```

Comment:
I had prison has its own district so it shows up on deaths by district etc.

Reply:
It doesn't show up on the `DF/df.columns` you posted earlier: https://github.com/djay/covidthailand/pull/20#issuecomment-851102723

What change do you propose?

Reply:
Oh, do you mean the `Prison` entry in the `province_mapping.csv` has it's own  `Health District Number` cell value as `'Health District Number'='Prison'`?

Reply:
If so, just edit the CSV file. - That's the nice thing about making this strategy to make the data more easily manageable.

Reply:
Correct. That was in teh code that was replaced.

Reply:
Replaced that entry in the CSV file.